pkg/cmd/cli/cli.go

```diff
@@ -0,0 +1,81 @@
+package cli
+
+import (
+	"flag"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	utilglog "github.com/openshift/source-to-image/pkg/util/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/source-to-image/pkg/api"
+	"github.com/openshift/source-to-image/pkg/cmd/cli/cmd"
+	cmdutil "github.com/openshift/source-to-image/pkg/cmd/cli/util"
+	"github.com/openshift/source-to-image/pkg/docker"
+)
+
+// glog is a placeholder until the builders pass an output stream down
+// client facing libraries should not be using glog
+var glog = utilglog.StderrLog
+
+// NewCmdCLI implements the S2I command line functionality.
+func NewCmdCLI(cfg *api.Config) *cobra.Command {
```

Comment:
what's the point of this arg?  seems like we only pass nil when we call NewCmdCLI....